cmake/developer_package/features.cmake

```diff
@@ -58,7 +58,11 @@ ie_option (VERBOSE_BUILD "shows extra information about build" OFF)
 
 ie_option (ENABLE_UNSAFE_LOCATIONS "skip check for MD5 for dependency" OFF)
 
-ie_dependent_option (ENABLE_FUZZING "instrument build for fuzzing" OFF "CMAKE_CXX_COMPILER_ID MATCHES ^(Apple)?Clang$; NOT WIN32" OFF)
+if(CMAKE_CXX_COMPILER_ID MATCHES "^(Apple)?Clang$")
+    set(IS_CLANG ON)
+endif()
+
+ie_dependent_option (ENABLE_FUZZING "instrument build for fuzzing" OFF "IS_CLANG; NOT WIN32" OFF)
 
```

Comment:
fix is provided here https://github.com/openvinotoolkit/openvino/pull/6722 

Please, revert changes in this file

Reply:
fixed